smoke-test/README.md

```diff
@@ -6,9 +6,11 @@ We currently leverage Docker and the TestContainers framework for running the su
 
 ## Running the system tests manually
 
-The tests require Docker images to run.
 
-### A) Pull existing images from DockerHub
+### Getting the Docker Images
+The tests require Docker images to run. There are two alternatives to get them, a) or b):
+
+#### a) Pull existing images from DockerHub
```

Comment:
Can we make these changes in develop and revert here? Otherwise we will face merge conflicts all the time, because this will probably evolve much faster in develop than in this branch.

Reply:
makes sense, I move them.